src/dmd/expression.d

```diff
@@ -6161,6 +6162,7 @@ extern (C++) final class UshrAssignExp : BinAssignExp
  */
 extern (C++) class CatAssignExp : BinAssignExp
 {
+    bool isCTFEHookPart;
```

Comment:
This has not been updated in the headers.

Reply:
I've added it to `expression.h`.